server/modlog.ts

```diff
@@ -497,59 +500,60 @@ export class Modlog {
 		if (rooms.includes('global')) roomChecker = `(roomid LIKE 'global-%' OR ${roomChecker})`;
 
 		let query = `SELECT *, (SELECT group_concat(userid, ',') FROM alts WHERE alts.modlog_id = modlog.modlog_id) as alts FROM modlog`;
+		if (this.useFTS) query += ` LEFT JOIN modlog_fts ON modlog_fts.rowid = modlog_id`;
```

Comment:
Any reason for using a left join here over a regular join?

Reply:
because modlog entries with no note won't have a row in the FTS table